viewer/src/components/LdTitle.vue

```diff
@@ -0,0 +1,46 @@
+<!-- ldgallery - A static generator which turns a collection of tagged
+--             pictures into a searchable web gallery.
+--
+-- Copyright (C) 2019-2020  Guillaume FOUET
+--
+-- This program is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU Affero General Public License as
+-- published by the Free Software Foundation, either version 3 of the
+-- License, or (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Affero General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
+-- along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-->
+
+<script lang="ts">
+import { Component, Vue, Watch, Prop } from "vue-property-decorator";
+
+@Component
+export default class LdTitle extends Vue {
+  @Prop({ required: true }) readonly galleryTitle!: string;
+  @Prop() readonly currentItem?: Gallery.Item;
+
+  render() {
+    return null;
+  }
+
+  mounted() {
+    this.changedCurrentItemPath();
+  }
+
+  @Watch("currentItem")
+  changedCurrentItemPath() {
+    document.title = this.generateTitle();
+  }
+
+  generateTitle(): string {
+    if (this.currentItem?.title) return `${this.galleryTitle} • ${this.currentItem.title}`;
```

Comment:
```suggestion
    if (this.currentItem?.title) return `${this.currentItem.title} • ${this.galleryTitle}`;
```


Should swap both the page and global title.  There seem to be a [consensus about that](https://webmasters.stackexchange.com/q/16251).